### PR TITLE
Makes nav/toolbar behaviour consistent for Feeds, Timeline, and Article

### PR DESCRIPTION
**Pull request by stuartbreckenridge**

### CODE REVIEW

File: iOS/Article/ArticleViewController.swift [hunk 1]
```diff
@@ -229,6 +230,27 @@ class ArticleViewController: UIViewController, MainControllerIdentifiable {
...
+		let standard = UINavigationBarAppearance()
+		standard.shadowColor = .opaqueSeparator
+		standard.shadowImage = UIImage()
+		
+		navigationController?.navigationBar.standardAppearance = standard
+		navigationController?.navigationBar.compactAppearance = standard
+		navigationController?.navigationBar.scrollEdgeAppearance = scrollEdge
+		navigationController?.navigationBar.compactScrollEdgeAppearance = scrollEdge
+	}
+
```
Comment (vincode-io) [inline iOS/Article/ArticleViewController.swift:252]: Is it possible to move this code to `SceneCoordinator.configureNavigationController` or adjacent to it?

Comment (stuartbreckenridge) [reply iOS/Article/ArticleViewController.swift]: Fixed in 90b4199

**Comment by vincode-io** [comment]
When I look at `MasterTimelineViewController`, I still see the `configureNavbar` function. Doesn't the change to `SceneCoordinator` handle this?

**Comment by stuartbreckenridge** [comment]
@vincode-io It's definitely removed in 90b4199 
Screenshot 2022-02-07 at 15 16 33

**Comment by vincode-io** [comment]
That was weird. I was clearly showing it when I directly viewed you PR, but it wasn't there when I merged it. Sorry for the confusion.

**Pull request merged**